browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2251,6 +2251,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 					button.setAttribute('aria-expanded', true);
 				}
 			}
+			builder.map.fire('unotoolbuttonclick', {command: data.command});
```

Comment:
Please do not add separate interceptor for single command detection in generic code for all the buttons (every button will call it for no reason mostly).

We already have place in the code where goes any uno command called, before it is sent to the server:
I'm not saying that recommended place is perfect but at least is the centralized one already :)

https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Toolbar.js#L356

Reply:
The issue is that "sendUnoCommand" is a generic function and does not guarantee that it is triggered by a user’s click event. The goal is to notify the system when a user clicks on the notebook-bar. Do you know where I can find it?

Reply:
I think if uno command is called by JSDialog component (eg. notebookbar) it is warented to do there

Reply:
see default handler for all user input: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.JSDialogBuilder.js#L256

Reply:
Alright, I will make the changes to the default JSDialog handler.

Reply:
Please do in the `sendUnoCommand` directly, handler uses that, but remember other (non-jSDialog) code might also trigger that. So best to have single point where uno command interception happens.